### DISCUSSION TITLE
FV3_input_data_INCCN_aeroclim on AWS?

### DISCUSSION THREAD

**Discussion by benjamin-cash** [NOAA-EMC/global-workflow#2813] [Q&A] [answered]
Are the files for `aeroclim.m??.nc` available via AWS? It appears that the directory in the title is where they usually live, but they are not in the copy of the fix files I downloaded. Thanks!

**Comment by KateFriedman-NOAA** [comment, marked as answer]
@benjamin-cash Do you mean the `merra2[C].aerclim.2003-2014.m##.nc` fix files? If so, they are available in the primary fix set that is staged on AWS here: https://noaa-nws-global-pds.s3.amazonaws.com/index.html#fix/aer/20220805/

**Reply by benjamin-cash** [reply]
Thanks! I'm working from a run directory @pjpegion set me up with, so there is some occasional detective work required when files change names.
